apps/api/util/setup.sh

```diff
@@ -38,11 +39,6 @@ echo -ne "Installing yarn"
 npm install -g yarn > /dev/null 2>&1
 echo " done"
 
-echo -ne "Setting babashka preloads"
-BABASHKA_PRELOADS='(def lines (babashka.fs/read-all-lines "../.env"))'
-BABASHKA_PRELOADS=$BABASHKA_PRELOADS' (def env (into (sorted-map) ( map #(str/split % #"=") lines )))'
```

Comment:
we'll have to fix this in the future. Might be able to move to nbb which has better support with nodejs

Reply:
yes that's why i removed it